zaza/openstack/charm_tests/glance/tests.py

```diff
@@ -222,3 +223,46 @@ def test_100_create_delete_image(self):
         )
         self.assertEqual(image["size"], response["ContentLength"])
         openstack_utils.delete_image(self.glance_client, image["id"])
+
+
+class GlanceCinderBackendTest(test_utils.OpenStackBaseTest):
+    """Encapsulate glance tests using cinder backend."""
+
+    @classmethod
+    def setUpClass(cls):
+        """Run class setup for running glance tests with cinder backend."""
+        super(GlanceCinderBackendTest, cls).setUpClass()
+        cls.glance_client = openstack_utils.get_glance_session_client(
+            cls.keystone_session)
+        cls.cinder_client = openstack_utils.get_cinder_session_client(
+            cls.keystone_session)
+
+    def test_100_create_delete_image(self):
+        """Create an image and do a simple validation of it.
+
+        Validate the size of the image in both Glance API and Cinder API.
+        """
+        image_name = "zaza-cinder-test-image"
+        openstack_utils.create_image(
+            glance=self.glance_client,
+            image_url=openstack_utils.find_cirros_image(arch="x86_64"),
+            image_name=image_name,
+            backend="cinder",
+        )
+        images = openstack_utils.get_images_by_name(
+            self.glance_client, image_name)
+        self.assertEqual(len(images), 1)
+        image = images[0]
+
+        volume_name = 'image-'+image["id"]
```

Comment:
This ought to fail pep8 - no spaces between the operator.  It would probably be better as:

```python
volume_name = f'image-{image["id"]}'
# or
volume_name = 'image-{}'.format(image["id"])
```

I'm trying to remember if we still support xenial (python3.5) in zaza - we probably do.  In which case the f-string won't work and it'll need to be the `.format()` version.

Reply:
Done